lib/galaxy/webapps/galaxy/api/dynamic_tools.py

```diff
@@ -19,6 +24,48 @@
 DatabaseIdOrUUID = Union[DecodedDatabaseIdField, str]
 
 
+@router.cbv
+class UnprivilegedToolsApi:
+    # Almost identical to dynamic tools api, but operates with tool ids
+    # and is scoped to to individual user and never adds to global toolbox
+    dynamic_tools_manager: DynamicToolManager = depends(DynamicToolManager)
+
+    @router.get("/api/unprivileged_tools")
```

Comment:
I feel like I would prefer a name like 'user_tools' but if it is a big change - feel free to ignore. Also sorry I'm reviewing by commit so if any of these changes are undone in future commits feel free to ignore.

Also like the last comment - feel free to just dump the dynamic tools API endpoint and replace it with this. I think I like that name better also.

Reply:
Absolutely open to changing how we name things, we did already have 3 or 4 different versions when we hacked on this in Berlin. I'll keep that for a last pass when it's nearing completion though.